js/src/personal/amanda/Amanda.page.tsx

```diff
@@ -1,34 +1,45 @@
-import { Title, Text } from '@mantine/core'
+import { Button, Center, Title } from '@mantine/core'
+import { useViewportSize } from '@mantine/hooks'
+import { useState } from 'react'
 import styles from './Amanda.module.css'
+import { AmandaMainPage } from './Amanda.main.page.tsx'
 
 export function AmandaPage() {
+  const { height, width } = useViewportSize()
+  const [landingPage, setLandingPage] = useState(1)
```

Comment:
Would be easier to read if you made an enum for each page instead of using an int.